clang/include/clang/Basic/AttrDocs.td

```diff
@@ -1825,6 +1825,26 @@ be applied multiple times to the same loop.
   }];
 }
 
+def SYCLDeviceIndirectlyCallableDocs : Documentation {
+  let Category = DocCatFunction;
+  let Heading = "intel::device_indirectly_callable";
+  let Content = [{
+This attribute can only be applied to a functions and indicates that the
```

Comment:
"can only be applied to a functions" -> "can only be applied to functions" or perhaps "can only be applied to a function"

Reply:
Fixed